analysis/codelists.py

```diff
@@ -2,7 +2,7 @@
 
 
 f2f_consultation = codelist_from_csv(
-    "codelists/user-KatieDavies_1234-f2f-consultation.csv",
+    "codelists/user-KatieDavies_1234-generic-consultation.csv",
```

Comment:
I'm not sure why you renamed this from f2f to generic but I'm sure there is a good reason.

Reply:
the slight disadvantage of the name change is that we cant easily see the codes that changed in the git differences (like we can see in the virtual consultations).